revolv/templates/project/project.html

```diff
@@ -31,10 +31,7 @@ <h1 class="title-h1">{{ project.title }}</h1>
       <div class="container">
         <div class="funded-round">
           <span class="status-text">
-            <span class="round-depict">
-              <span>OF</span> 
-              ${{ project.funding_goal|floatformat:0|intcomma }}
-            </span>
+            <span class="round-depict"><span>OF</span>${{ project.funding_goal|floatformat:0|intcomma }}</span>
```

Comment:
There needs to be one space after this `span`: otherwise, it will show up as `OF$32,000` instead of `OF $32,000`